### ISSUE TITLE
Rails 5.1: Index Creation fails

### ISSUE THREAD

**Issue by drernie** [code-mancers/rapidfire#101]
I kept getting errors  on 'db:migrate' like:
```
-- add_index(:rapidfire_questions, :survey_id)
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

Index name 'index_rapidfire_questions_on_survey_id' on table 'rapidfire_questions' already exists
```
I ended up commenting out all the add_index lines.

**Comment by iffyuva** [comment]
Thanks for the report @drernie We will look into this.

**Comment by drernie** [comment]
Thanks.  FYI, I've also submitted a PR with all my changes for 5.1, in case
it helps.

**Comment by aksh93** [comment]
This issue has been resolved [#118](https://github.com/code-mancers/rapidfire/pull/118).
